kratos/includes/node.h

```diff
@@ -919,6 +929,11 @@ class Node : public Point,  public IndexedObject, public Flags
         return mSolutionStepsNodalData.pGetVariablesList();
     }
 
+    const VariablesList * pGetVariablesList() const
+    {
+        return mSolutionStepsNodalData.pGetVariablesList();
+    }
+
 
     /// TODO: remove this function when removing data_file_io object.
 //        IndexType& DepricatedIdAccess()

```

Comment:
we could take out this... anyway it is commented so it is a good time for cleanup